commTypes.h

```diff
@@ -17,7 +17,15 @@ enum class NetworkType { unknown = 0, generic, ssl, vss };
  * Type of the packet position.
  *  Attention: Limited in 8
  */
-enum class PositionType { unknown = 0, source, stop, motionControl, rotateControl, rotateInPoint };
+enum class PositionType {
+  unknown = 0,
+  source,
+  stop,
+  motionControl,
+  rotateControl,
+  rotateInPoint,
+  newMotionControl
```

Comment:
I think it would be better to use a more descriptive nomenclature, something like `globalMotionControl`

Reply:
Good call. This name was temporary and forgot to remove it when creating the PR. Will update it and leave it as it is already here.